CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
+## Version 2.5.0-beta1
+- Method `Sanitize` on classes implementing `ITelemetry` does not modify the `TelemetryContext` fields any longer. Serailized event json and ETW event will still have context tags sanitized.
```

Comment:
Suggest:
Method `Sanitize` on classes implementing `ITelemetry` no longer modifies the `TelemetryContext` fields.

Also: Typo: Serailized -> Serialized